src/Apps/__stories__/AppShell.story.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+
+import { SystemContextProvider } from "Artsy/SystemContext"
+import { MockRouter } from "DevTools"
+
+import { getAppRoutes } from "Apps/getAppRoutes"
+
+storiesOf("Apps", module).add("AppShell", () => {
+  return (
+    <SystemContextProvider>
+      <MockRouter
+        routes={getAppRoutes()}
+        initialRoute="/collect"
+        context={{
+          // FIXME: Remove when a/b test is complete
+          // @ts-ignore
+          COLLECTION_HUBS: "experiment",
```

Comment:
Is this a/b test still running?

Reply:
This is the entrypoint to the story, it was probably 🍝 from the collect story when it was running! Will clean up.